src/Driver.php

```diff
@@ -216,7 +216,7 @@ public function get(string $path): Promise;
      *
      * @param string $path The file path to which to $contents should be written
      * @param string $contents The data to write to the specified $path
-     * @return \Amp\Promise A promise resolving to the integer length written upon success
+     * @return Promise<int> A promise resolving to the integer length written upon success
```

Comment:
I guess this should also return `Promise<void>`

Reply:
Why? `A promise resolving to the integer length written upon success` - you would lose that information. Though maybe it's just always the same as `strlen($contents)` and that's why you wanna drop it?

Reply:
Yes, exactly, it's just `strlen` on the input.